tiles-request-api/src/main/java/org/apache/tiles/request/locale/URLApplicationResource.java

```diff
@@ -44,10 +42,45 @@
 public class URLApplicationResource extends PostfixedApplicationResource {
     private static final Logger LOG = LoggerFactory.getLogger(URLApplicationResource.class);
 
+    /**
+     * System parameter to specify additional remote protocols. If a url has a remote protocol, then any
+     * {@link IOException} will be thrown directly. If a url has a local protocol, then any {@link IOException}
+     * will be caught and transformed into a {@link FileNotFoundException}.
+     */
+    static final String REMOTE_PROTOCOLS_PROPERTY = "tiles.remoteProtocols";
```

Comment:
Trivial:
 according to Java's declaration order, this should appear before `LOG` is declared.